try:
	raise Exception("My exception")
except Exception:
	print("smth went wrong")